MatrixSDK/Data/RoomSummaryStore/CoreData/MXCoreDataRoomSummaryStore.xcdatamodeld/MXRoomSummaryCoreDataStore2.xcdatamodel/contents

```diff
@@ -62,10 +63,4 @@
         <attribute name="s_usersCount" attributeType="Integer 32" defaultValueString="0" usesScalarValueType="YES"/>
         <relationship name="s_ofRoom" optional="YES" maxCount="1" deletionRule="Nullify" destinationEntity="MXRoomSummary" inverseName="s_trust" inverseEntity="MXRoomSummary"/>
     </entity>
-    <elements>
-        <element name="MXRoomLastMessage" positionX="117" positionY="90" width="128" height="149"/>
-        <element name="MXRoomMembersCount" positionX="297.1484375" positionY="-66.14453125" width="128" height="89"/>
-        <element name="MXRoomSummary" positionX="-63" positionY="-18" width="128" height="509"/>
-        <element name="MXUsersTrustLevelSummary" positionX="236.4921875" positionY="519.4296875" width="128" height="104"/>
-    </elements>
```

Comment:
I think this has been removed automatically by Xcode since they removed the [Core Data graph view](https://stackoverflow.com/questions/72824184/where-does-xcodes-core-data-graph-goes)! 😭

Reply:
Nooooo, why did they do that?! 😢